src/libZilliqa/Zilliqa.cpp

```diff
@@ -61,6 +62,20 @@ void Zilliqa::LogSelfNodeInfo(const std::pair<PrivKey, PubKey>& key,
                                      << peer.m_listenPortHost);
 }
 
+/*static*/ std::string Zilliqa::FormatMessageName(unsigned char msgType,
+                                                  unsigned char instruction) {
+  const std::string InvalidMessageType = "INVALID_MESSAGE";
+  if (msgType >= ARRAY_SIZE(MessageTypeStrings)) return InvalidMessageType;
+
+  if (NULL == MessageTypeInstructionStrings[msgType]) return InvalidMessageType;
+
+  if (instruction >= MessageTypeInstructionSize[msgType])
```

Comment:
You may want to keep the every statement for if and else in  {} as per our standard format.